src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -811,7 +811,7 @@ def load_arguments(self, _):
                           "To enable Azure Hybrid Benefit for Windows Server, use 'Windows_Server'. " \
                           "To enable Multitenant Hosting Rights for Windows 10, use 'Windows_Client'. " \
                           "For more information see the Azure Windows VM online docs."
-            c.argument('license_type', help=license_msg, arg_type=get_enum_type(['Windows_Server', 'Windows_Client', 'None']))
+            c.argument('license_type', help=license_msg, arg_type=get_enum_type(['Windows_Server', 'Windows_Client', 'RHEL_BYOS', 'SLES_BYOS', 'None']))
```

Comment:
No enum defined in  SDK ?

Reply:
No. It is a string type in swagger.

Reply:
some general question.
- for this kind of string type enum in Swagger, how about file an issue in swagger repo to service team to get awareness?
- case insenstive/sensitive rule at CLI side?  Or we just pass in what service required?

Reply:
1. They know it. They don't declare they will fix it. I am OK with string type.
2. Enum type in CLI supports case insensitive matching.